README.md

```diff
@@ -252,3 +252,45 @@ python3 -m twine upload dist/*
 
 ## Requirements
 Python 3.9 or newer
+
+# Optional Timeout and Payload Size Features
+As a Flask application, the ubkg-api can be deployed in various environments.
+
+When deployed behind a server gateway, such as AWS API Gateway, the gateway may impose constraints
+on timeout or response payload size. For example, AWS API Gateway has a timeout of 29 seconds and
+a maximum response payload of 10 MB.
+
+The ubkg-api can handle timeouts and large payloads before they result in errors in a gateway. 
+The ubkg-api will return detailed explanations for timeout or payload problems, instead of relying on the 
+sometimes ambiguous messages from the gateway (e.g., a HTTP 500).
+
+To enable custom management of timeout and payload size, specify values in the **app.cfg** file, as shown below.
+
+```commandline
+# Maximum duration for the execution of timeboxed queries to prevent service timeout, in seconds
+# The AWS API gateway timeout is 29 seconds.
+TIMEOUT=28
+# Maximum size of payload, in bytes. This is designed to accomodate limits set by an AWS API gateway, which
+# has a 10 MB payload limit for the AWS API gateway. A value of 0 means that there is no payload limit.
+PAYLOADLIMIT=9*(2**20) + 900*(2**10) #9.9Mb
```

Comment:
In favor of the new S3 handling with using `LARGE_RESPONSE_THRESHOLD`, I think we should remove the old `PAYLOADLIMIT` from and config and the codebase.